src/app/landing/index.tsx

```diff
@@ -44,10 +44,12 @@ export default function Landing() {
       <GridItem display={{ base: 'none', xl: 'initial' }} colSpan={1}>
         <Container>
           <Image
+            alt=""
             src={process.env.PUBLIC_URL + '/assets/computer.svg'}
             sx={{
               filter: 'drop-shadow( 1.5rem 1rem 1.75rem rgba(0, 0, 0, .25) )',
             }}
+            
```

Comment:
There is a linting error here - there shouldn't be an empty line.
If you run `npm run lint:fix` it should fix it 😄